### PR TITLE
chore: Optimize logging changing LOGs to VLOGs

### PR DESCRIPTION
**Pull request by BagritsevichStepan**
relates to [the comment](https://github.com/dragonflydb/dragonfly/pull/3285#discussion_r1669959450). 

Temporary solution. Still looking for places in the code where we need to change `LOG` to `VLOG`

### CODE REVIEW

**Comment by kostasrim** [comment]
@BagritsevichStepan if you find any other cases like that in the hot path it would be great to clean them :) 

Thank you again

File: src/server/search/doc_accessors.cc [hunk 1]
```diff
@@ -213,7 +213,7 @@ JsonAccessor::JsonPathContainer* JsonAccessor::GetPath(std::string_view field) c
   }
 
   if (!ptr) {
-    LOG(WARNING) << "Invalid Json path: " << field << ' ' << ec_msg;
+    VLOG(2) << "Invalid Json path: " << field << ' ' << ec_msg;
```
Comment (romange) [inline src/server/search/doc_accessors.cc:216]: level 1 is enough, it's pretty uncommon so low chance of flooding the logs.

**Review by adiholden — APPROVED** [review]

**Comment by romange** [comment]
@BagritsevichStepan is it still relvant? if not pls close

**Comment by dranikpg** [comment]
Let's change it to a dcheck of keep it as is

**Pull request closed without merging**